tests/test_integration/test_load_data.py

```diff
@@ -26,3 +26,6 @@ def test_load_dataset_from_csv():
     )
     ds_from_json = import_fluxomics_dataset_from_json(MODEL_FILE)
     assert ds_from_csv == ds_from_json
+
+
+test_load_dataset_from_csv()
```

Comment:
This line isn't necessary if you run tests with pytest: it should automatically run any function with `test` in its name.